evennia/web/website/forms.py

```diff
@@ -0,0 +1,142 @@
+from django import forms
+from django.conf import settings
+from django.contrib.auth.forms import UserCreationForm, UsernameField
+from evennia.utils import class_from_module
+from random import choice, randint
+
+class AccountCreationForm(UserCreationForm):
+    
+    class Meta:
+        model = class_from_module(settings.BASE_ACCOUNT_TYPECLASS)
+        fields = ("username", "email")
+        field_classes = {'username': UsernameField}
+    
+    email = forms.EmailField(help_text="A valid email address. Optional; used for password resets.", required=False)
+    
+class CharacterCreationForm(forms.Form):
+    name = forms.CharField(help_text="The name of your intended character.")
+    age = forms.IntegerField(min_value=3, max_value=99, help_text="How old your character should be once spawned.")
+    description = forms.CharField(widget=forms.Textarea(attrs={'rows': 3}), max_length=2048, min_length=160, required=False)
+    
+    @classmethod
+    def assign_attributes(cls, attribute_list, points, min_points, max_points):
+        """
+        Randomly distributes a number of points across the given attributes,
+        while also ensuring each attribute gets at least a certain amount
+        and at most a certain amount.
+        
+        Args:
+            attribute_list (iterable): List or tuple of attribute names to assign
+                points to.
+            points (int): Starting number of points
+            min_points (int): Least amount of points each attribute should have
+            max_points (int): Most amount of points each attribute should have
+            
+        Returns:
+            spread (dict): Dict of attributes and a point assignment.
+            
+        """
+        num_buckets = len(attribute_list)
+        point_spread = (x for x in self.random_distribution(points, num_buckets, min_points, max_points))
+        
+        # For each field, get the point calculation for the next attribute value generated
+        return {attribute: next(point_spread) for k in attribute_list}
+    
+    @classmethod
+    def random_distribution(cls, points, num_buckets, min_points, max_points):
```

Comment:
Same here, this does not belong in core. If wanting to offer these functions, they rather belong as utilities that the user can pick and chose from.